### ISSUE TITLE
[Impeller] Crash due to invalid texture descriptor

### ISSUE THREAD

**Issue by siddharthadevops** [flutter/flutter#119818]
## Command
```
flutter --no-color run --machine --device-id=00008120-00062D983CC0C01E --profile --dart-define=flutter.inspector.structuredErrors=true lib/main.dart
```

## Steps to Reproduce
1. Enable Impeller on iOS.

## Logs

[VERBOSE-3:validation.cc(31)] Texture descriptor was invalid.
* thread #15, name = 'io.worker.3', stop reason = signal SIGABRT
    frame #0: 0x00000001da793160 libsystem_kernel.dylib`__pthread_kill + 8
libsystem_kernel.dylib`:
->  0x1da793160 <+8>:  b.lo   0x1da793180               ; <+40>
    0x1da793164 <+12>: pacibsp 
    0x1da793168 <+16>: stp    x29, x30, [sp, #-0x10]!
    0x1da79316c <+20>: mov    x29, sp
Target 0: (Runner) stopped.

```

```
```
[✓] Flutter (Channel stable, 3.7.1, on macOS 13.2 22D49 darwin-arm64, locale es-ES)
    • Flutter version 3.7.1 on channel stable at /Users/siddhartha/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7048ed95a5 (19 hours ago), 2023-02-01 09:07:31 -0800
    • Engine revision 800594f1f4
    • Dart version 2.19.1
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/siddhartha/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14C18
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[✓] IntelliJ IDEA Community Edition (version 2022.3.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 71.3.6
    • Dart plugin version 223.8617.8

[✓] VS Code (version 1.70.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.54.0

[✓] Connected device (3 available)
    • iPhone de Dante (mobile) • 00008120-00062D983CC0C01E • ios            • iOS 16.2 20C65
    • macOS (desktop)          • macos                     • darwin-arm64   • macOS 13.2 22D49 darwin-arm64
    • Chrome (web)             • chrome                    • web-javascript • Google Chrome 109.0.5414.119
    ! Error: (null) needs to connect to determine its availability. Check the connection between the device and its companion iPhone, and the connection between the iPhone and Xcode. Both devices may also need to be restarted and unlocked. (code 1)

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+49
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 18116933e77adc82f80866c928266a5b4f1ed645
### Plugins
app_group_directory-2.0.0
audio_service-0.18.9
audio_service_web-0.1.1
audio_session-0.1.13
audioplayers-3.0.1
audioplayers_android-2.0.0
audioplayers_darwin-3.0.1
audioplayers_linux-1.0.4
audioplayers_web-2.2.0
audioplayers_windows-1.1.3
background_fetch-1.1.4
camera-0.10.3
camera_android-0.10.3
camera_avfoundation-0.9.11
camera_web-0.3.1+1
connectivity_plus-3.0.2
country_codes-2.2.0
eraser-2.0.1
file_picker-5.2.5
firebase_auth-4.2.5
firebase_auth_web-5.2.4
firebase_core-2.4.1
firebase_core_web-2.1.0
firebase_crashlytics-3.0.11
firebase_messaging-14.2.1
firebase_messaging_web-3.2.11
firebase_storage-11.0.10
firebase_storage_web-3.3.19
flutter_app_badger-1.5.0
flutter_contacts-1.1.5+1
flutter_image_compress-1.1.3
flutter_local_notifications-13.0.0
flutter_native_splash-2.2.16
flutter_plugin_android_lifecycle-2.0.7
flutter_secure_storage-7.0.1
flutter_secure_storage_linux-1.1.2
flutter_secure_storage_macos-2.0.1
flutter_secure_storage_web-1.1.1
flutter_secure_storage_windows-1.1.3
geocoding-2.0.5
geolocator-9.0.2
geolocator_android-4.1.7
geolocator_apple-2.2.5
geolocator_web-2.1.6
geolocator_windows-0.1.1
google_maps_flutter-2.2.3
google_maps_flutter_android-2.4.3
google_maps_flutter_ios-2.1.13
image_cropper-3.0.1
image_cropper_for_web-1.0.3
image_picker-0.8.6+1
image_picker_android-0.8.5+5
image_picker_for_web-2.1.10
image_picker_ios-0.8.6+6
just_audio-0.9.31
just_audio_web-0.4.7
lecle_downloads_path_provider-0.0.2+8
libphonenumber_plugin-0.2.5
libphonenumber_web-0.2.0+1
map_launcher-2.4.0
maps_launcher-2.0.1
objectbox_flutter_libs-1.7.2
path_provider-2.0.12
path_provider_android-2.0.22
path_provider_foundation-2.1.1
path_provider_linux-2.1.7
path_provider_windows-2.1.3
permission_handler-10.2.0
permission_handler_android-10.2.0
permission_handler_apple-9.0.7
permission_handler_windows-0.1.2
phone_number-1.0.0
photo_manager-2.5.2
r_album-0.1.4
record-4.4.4
record_linux-0.4.1
record_macos-0.2.2
record_web-0.5.0
record_windows-0.7.1
share_plus-6.3.0
sound_mode-2.0.2
soundpool-2.3.0
soundpool_macos-2.2.0
soundpool_web-2.2.0
sqflite-2.2.4+1
syncfusion_flutter_pdfviewer-20.4.48
syncfusion_pdfviewer_macos-20.4.48
syncfusion_pdfviewer_web-20.4.48
syncfusion_pdfviewer_windows-20.4.48
url_launcher-6.1.8
url_launcher_android-6.0.23
url_launcher_ios-6.0.18
url_launcher_linux-3.0.2
url_launcher_macos-3.0.2
url_launcher_web-2.0.14
url_launcher_windows-3.0.3
video_compress_ds-3.1.6
video_player-2.5.1
video_player_android-2.3.10
video_player_avfoundation-2.3.8
video_player_web-2.0.13
volume_controller-2.0.6
wakelock-0.6.2
wakelock_macos-0.4.0
wakelock_web-0.4.0

**Comment by exaby73** [comment]
I cannot reproduce this issue on my machine. Labelling for further insights from the team

**Comment by jonahwilliams** [comment]
We'd need to know which image is trying to be loaded to identify the problem - the validation error could be related to the mip level calculation we fixed with 1x1 images, or it could be unrelated. See https://github.com/flutter/flutter/issues/119245

**Comment by siddharthadevops** [comment]
Hello, I've removed a widget that was using kTransparentImage and the problem was solved. So the issue is the same as https://github.com/flutter/flutter/issues/119245.

**Comment by exaby73** [comment]
@siddharthadevops Looks like that issue is fixed. Could you try using `kTransparentImage` with the master channel and confirm?

**Comment by siddharthadevops** [comment]
I was unable to test this issue on the master branch due to several incompatibilities with some packages. But I can confirm that the problem it's solved in the latest release of Flutter 3.7.3

**Comment by exaby73** [comment]
Thank you for the information :) Based on your comment, I feel safe to close this issue. Thank you
